climpred/comparisons.py

```diff
@@ -1,9 +1,20 @@
+import dask
 import numpy as np
 import xarray as xr
 
 from .checks import has_dims, has_min_len
 from .exceptions import DimensionError
 
+# import error
```

Comment:
I will open an issue for those kinds of errors. a software architecture question